### ISSUE TITLE
Suppress notices from cbox-theme

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/commons-in-a-box#115]
cbox-theme throws a bunch of strict standards notices. @r-a-y suggested that we suppress them. Let's do it.

**Comment by r-a-y** [comment]
I decided to fix the notices wherever possible in cbox-theme.

The WPThumb notice could probably be fixed by upgrading WPThumb, which is a 3rd-party plugin by Human Made, but I didn't want to go through another round of testing, so I opted for the simple fix.

The only place I'm suppressing notices is for the `'bp_setup_current_user'` / `'bbp_setup_current_user'` notices in https://github.com/cuny-academic-commons/cbox-theme/commit/0688e61719dec3199e681461a308636aab12f501 since I didn't want to mess with ICE's load order.

Did some testing and didn't break anything, which is always a good sign 😄

The cbox-theme dashboard now loads without having to set `WP_DEBUG` to `false`.
